qiita_pet/handlers/admin_processing_job.py

```diff
@@ -91,3 +92,40 @@ def get(self):
 
         # return the json in compact form to save transmit size
         self.write(dumps(results, separators=(',', ':')))
+
+
+class SampleValidation(AdminProcessingJobBaseClass):
+    @coroutine
+    @execute_as_transaction
+    def get(self):
+        self._check_access()
+
+        self.render("sample_validation.html", input=True)
+
+    @execute_as_transaction
+    def post(self):
+
```

Comment:
I would consider removing this blank line at the top. Some coders begin functions and methods immediately after the declaration and some don't. However most if not all code in Qiita begins immediately after and it's good to be consistent.

Reply:
I believe the blank line was recommended by flake8 - let me know if you would still like me to remove it

Reply:
I don't recall flake8 doing that. Yes please.

Reply:
Removed the blank line